server/app.es6

```diff
@@ -14,9 +16,23 @@ let app = express();
 /* Use helmet for increased security. */
 app.use(helmet());
 
+/* Download login.gov cert. */
+if (process.env.PLATFORM !== 'CircleCI') {
+  try {
+    util.prepareCerts();
```

Comment:
I feel like I'd be sort of wary of just pulling down the key to the filesystem app server start. I'd rather have the stream pulled down as a promise to in memory at the time of login,  as I don't think there would be much of a performance hit, but I think it would be more secure.